### PR TITLE
feat(setting): Improved functionality of desk list

### PR DESCRIPTION
**Pull request by pavlovicnemanja92**

### CODE REVIEW

**Comment by amagdas** [comment]
:+1:

**Comment by Aca-jov** [comment]
:+1:

**Comment by mdhaman** [comment]
:+1:

File: client/app/scripts/superdesk-desks/views/settings.html [hunk 1]
```diff
@@ -1,37 +1,64 @@
...
-	    <div class="content desk-settings">
-	    	<div sd-shadow>
-	    		<ul class="pills-list">
-					<li ng-repeat="desk in desks._items">
-					
-						<h6 class="pull-left">{{ desk.name }}</h6>
+        <div class="header">
+            <h2 translate>Desk management</h2>
+            <button class="btn btn-info pull-right" ng-click="openDesk('general')">
+                <i class="icon-plus-sign icon-white"></i> {{ 'Add New Desk' | translate }}
```
Comment (petrjasek) [inline client/app/scripts/superdesk-desks/views/settings.html]: pls use one time binding for translate

File: client/app/scripts/superdesk-desks/views/settings.html [hunk 2]
```diff
@@ -1,37 +1,64 @@
...
 
-		<div sd-desk-config-modal 
-			data-active="modalActive" 
-			data-desk="desk" 
-			data-desks="desks" 
-			data-step="step"
-			data-cancel="cancel()"
-		></div>
-	</div>
+                    <div class="flex-item card-box" ng-repeat="desk in desks._items">
```
Comment (petrjasek) [inline client/app/scripts/superdesk-desks/views/settings.html]: pls add track by

**Pull request merged**